src/llmcompressor/transformers/compression/sparsity_config.py

```diff
@@ -181,7 +181,10 @@ def is_sparse24_bitmask_supported(
             return False
 
         if not is_model_quantized(model):
-            # non-quantized 2:4 sparse models are supported
+            logger.warning(
+                "Compressed Sparse-only 2:4 models are not supported in vLLM<=0.7.0, "
+                "consider saving with disable_sparse_compression=True"
```

Comment:
How does user pass `disable_sparse_compression` to this func?

Reply:
Yeah might be worth adding it's in the `.from_pretrained` input args

Reply:
Is this better?